src/OrchardCore.Modules/Orchard.OpenId/Startup.cs

```diff
@@ -42,6 +42,7 @@ public class Startup : StartupBase
 
         public override void Configure(IApplicationBuilder builder, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
+            /*
```

Comment:
Yes, as mentioned in my above list about `Orchard.OpenId`.

    Just commented out lines in the Startup to be able to build and run

So, for this part i didn't do anything 😄 

In my last comment, when i said looks good it was related to the last work of @alexbocharov.